src/models/slices/meta.ts

```diff
@@ -1,7 +1,20 @@
 import { Context } from '@yext/answers-core';
 
+/**
+ * Maintains the metadata for network requests.
+ */
 export interface MetaState {
+  /**
+   * A JSON object used for triggering and passing data to query rules.
```

Comment:
It might be helpful here to link to the official docs around `Context`. I think there is a page in the API documentation somewhere that explains what it is and the limitations. Rose could point you in the right direction.

Reply:
Added!